src/main/java/today/sesac/shoutify/post/entity/Post.java

```diff
@@ -0,0 +1,88 @@
+package today.sesac.shoutify.post.entity;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import today.sesac.shoutify.global.domain.BaseEntity;
+
+/**
+ * 게시판의 게시글을 관리하는 도메인입니다.
+ */
+@Entity //엔티티임을 표시
+@Table(name = "posts") //엔티티<->매핑할 테이블 , 이름 생략시 엔티티 이름 사용, 유니크 제약조건
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED) //매개변수 없는 생성자를 protected로 만들어줌
+
+public class Post extends BaseEntity {
+
+    @Id //기본키
+    @GeneratedValue(strategy = GenerationType.IDENTITY) //auto_increment
+    private Long id;
+    /**
+     * 비속어 ai 수정 전 게시물 내용입니다.
+     */
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String afterContents;
+    /**
+     * 게시물을 작성한 회원입니다
+     */
+//    @ManyToOne
+//    @JoinColumn(name = "author", nullable = false)
+//    private User user;
+    /**
+     * 게시물의 제목입니다.
+     */
+    @Column(length = 255)
+    private String title;
+    /**
+     * 게시물 작성후 ai로 판단되는 게시물에 대한 점수입니다.
+     */
+    @Column(nullable = false, columnDefinition = "INT DEFAULT 0")
+    private int score = 0;
+    /**
+     * 게시물 내의 이미지 입니다. url로 저장됩니다.
+     */
+    @Column(length = 500)
+    private String imageUrl;
+    /**
+     * 게시글 신고 유무입니다. 기본값 = 0
+     */
+    @Column(nullable = false, columnDefinition = "TINYINT DEFAULT 0")
+    private boolean isReported = false;
+    /**
+     * 게시글 삭제 유무입니다. soft deleted이므로 삭제되어도 db에는 존재하게됩니다.
+     * 기본값 = 0
+     */
+    @Column(nullable = false, columnDefinition = "TINYINT DEFAULT 0")
+    private boolean isDeleted = false;
+    /**
+     * 숨김 유무입니다. 기본값 = 0
+     */
+    @Column(nullable = false, columnDefinition = "TINYINT DEFAULT 0")
+    private boolean isHidden = false;
+    /**
+     * 자신의 게시물에 등록하는 이모지 입니다.
+     * null일 경우(작성자가 선택하지 않았을 경우) ai가 글의 이모지를 선택해줍니다.
+     */
+    @Column(length = 20)
+    private String my_emoji;
+
+//    private Post(User user, String afterContents, String title, Integer score, String imageUrl, String my_emoji) {
+//        this.user = user;
+//        this.afterContents = afterContents;
+//        this.title = title;
+//        this.score = score;
+//        this.imageUrl = imageUrl;
+//        this.isReported = false; //Boolean은 디폴트가 false
+//        this.isDeleted = false;
+//        this.isHidden = false;
+//        this.my_emoji = my_emoji;
```

Comment:
P3
이 부분은 나중에 기본값만 아래에서 따로 모아서 적어주면, 나중에 확인할 때 더 편할 것 같습니다~!

```suggestion
//        this.imageUrl = imageUrl;
//        this.my_emoji = my_emoji;
// 
//        // 기본값 설정
//        this.isReported = false; 
//        this.isDeleted = false;
//        this.isHidden = false;
```